src/backend/src/services/work-packages.services.ts

```diff
@@ -605,7 +606,7 @@ export default class WorkPackagesService {
 
     const upcomingWorkPackages = workPackages
       .map(workPackageTransformer)
-      .filter((wp) => daysBetween(wp.endDate, new Date()) <= daysUntilDeadline)
+      .filter((wp) => getDay(wp.endDate) <= getDay(deadline))
```

Comment:
why do we need to do it this way instead of just comparing dates?

Reply:
to avoid any weird time bugs
what if the end date is 12/25/xx at 3pm and then the deadline is 12/25 at 0:00 -- if we just compare times then a message wont send for that day even though it should